enterprise-suite/tests/timeout.fn

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Execute a command with a timeout
+
+# This has been modified so that it's a function instead of a command.
+# Also return 142 on timeout to match "kill SIGALARM" return value.
+
+# License: LGPLv2
+# Author:
+#    http://www.pixelbeat.org/
+# Notes:
+#    Note there is a timeout command packaged with coreutils since v7.0
```

Comment:
@marcoderama is there a reason not to use the coreutils timeout command?

Reply:
Can only depend on coreutils on linux. Mac OS, for example, doesn't have `timeout`.